docs/development/how-to/ide-setup/eclipse.md

```diff
@@ -53,5 +52,6 @@ Should look like this when configured: <br />
   - [EGit in Eclipse](http://www.eclipse.org/egit/) - with [tutorial
       ](http://www.vogella.com/tutorials/EclipseGit/article.html)
   - Run Configurations are checked in
-  - Run `./gradlew downloadAssets` one time to get images downloaded
-
+  - Run `./gradlew downloadAssets` (Windows: `gradlew.bat downloadAssets`) once in the main triplea folder to get images downloaded
+  - Run `./gradlew downloadAssets` (Windows: `gradlew.bat downloadAssets`) once in the main triplea folder to get images downloaded
+  - Run `./gradlew spotlessApply` (Windows: `gradlew.bat spotlessApply`) at least once in the main triplea folder if errors occur during gradle build for any of the tasks (it might have to be run multiple times though)
```

Comment:
I suspect this was due to running on windows, needing to run spotlessApply arguably is not related to the eclipse setup. 

Without knowing the exact errors you were hitting @frigoref , I suspect they were line ending problems. Those can be solved via a local config: https://docs.github.com/en/get-started/getting-started-with-git/configuring-git-to-handle-line-endings and would not require you to repeatedly run spotlessApply to fix.

Reply:
I do not understand what you mean, but I gut error messages saying "_Run gradlew :xxx:spotlessApply_" several times. First I ran them individually and then I just used _gradlew spotlessApply_

Reply:
- Needing to run 'spotlessApply (formatting) is not relevant to IDE setup. The formatting is fine after checkout.

- An exact error message would be helpful. I suspect the problem are line endings being silently converted when you save files. That is an OS and configuration problem and can be fixed.